survey_one_choice_per_column/views/assets.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 ONESTEiN BV (<http://www.onestein.eu>)
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<openerp>
+    <data>
+        <template id="survey_one_choice_per_column_assets" inherit_id="website.assets_frontend" name="Survey One Choice Per Column Assets">
```

Comment:
Can you improve a the format with shorter lines?
It helps the review in general and gets better readability